tox.ini

```
@@ -1,7 +1,5 @@
 [tox]
-isolated_build = True
-minversion = 3.20.0
-distshare = {homedir}/.tox/distshare
+requires = tox>=4
```

Let's make it sparse for readability.
```suggestion
requires =
  tox >= 4
```